static/css/nl_interface.scss

```diff
@@ -113,20 +113,29 @@ $answer-min-height: 100vh;
   }
 
   .bar-chart .legend,
-  .line-chart .legend {
+  .line-chart .legend,
+  .block-title {
     display: none;
   }
 
-  .block-title {
-    display: none;
+  .tile-hidden {
+    position: absolute;
+    top: -1000px;
```

Comment:
nit: could this just be `display: none`?

Reply:
with display: none, all the calls to getBBox() during chart drawing receives 0, so the charts never render. this way, they are rendered but out of the way